src/battle/other_battle_calculators.c

```diff
@@ -3296,3 +3296,879 @@ BOOL LONG_CALL ov12_02251A28(struct BattleSystem *bsys, struct BattleStruct *ctx
 
     return ret;
 }
+
+static BOOL LONG_CALL ov12_0224B528(struct BattleSystem *bsys, struct BattleStruct *ctx) {
+    int effect = ctx->moveTbl[ctx->current_move_index].effect;
+    int ret = 0;
+    
+    do {
+        switch (ctx->ssc_seq_no) {
+        case 0:
+            ctx->battlemon[ctx->attack_client].condition2 &= ~0x2000000;//STATUS2_DESTINY_BOND
```

Comment:
please feel free to fill out the STATUS2 constants by all means (same goes for berry flavors later down)

i can do it if you're not feeling up to it

Reply:
STATUS2_DESTINY_BOND isnt a constant included in battle.h (its used by ASM includes) so I didnt fill them in

Reply:
all the flavor stuff also arent defined